qwoyn/chain.json

```diff
@@ -0,0 +1,90 @@
+{
+  "$schema": "../chain.schema.json",
+  "chain_name": "qwoyn",
```

Comment:
if this is a testnet, please rename the folder and the chain_name in both files to qwoyntestnet, and move the folder into the /testnets/ folder